src/Controller/SecurityController.php

```diff
@@ -57,16 +57,17 @@ public function editUser(
             throw $this->createAccessDeniedException('vous devez etre connecté');
         }
         $userService = new UserService();
```

Comment:
Objects behind forms are not services.
Is there a reason not to use the User object this forms seems to edit ?